src/GameShelf.js

```diff
@@ -1,19 +1,27 @@
-import React from 'react';
+import React, { Component } from 'react';
 import GameCard from './GameCard';
 import './GameShelf.css';
 
-export default function Gameshelf(props) {
+export default class GameShelf extends Component {
+  constructor (props) {
+    super (props);
+    this.state = {}
+  } 
+  render () {
+    let filteredGames = this.props.filteredGames
```

Comment:
You can use ES6 destructuring here (some syntax you'll see more frequently as you look into more React examples) to accomplish the same thing:

`let { filteredGames } = this.props;`